dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/rest/service/DssRestActorFormDataService.java

```diff
@@ -29,26 +30,41 @@ public DssRestActorFormDataService(String name, String path, DssRestMethodType m
 
     @SuppressWarnings("unchecked")
     @Override
-    protected final HashMap<String, Object> getBody(String content) {
-        if (StringUtils.isEmpty(content)) {
+    protected final HashMap<String, Object> getBody(DssRestServiceActorCommandRequest commandRequest) {
+        if (StringUtils.isEmpty(commandRequest.getContent())) {
             return new HashMap<>();
         }
 
-        final String[] splits = content.split("&");
-        final HashMap<String, Object> bodyMap = new HashMap<>(splits.length);
-
-        for (int i = 0; i < splits.length; i++) {
-            final String element = splits[i];
-            if (!StringUtils.isEmpty(element)) {
-                final String[] formData = element.split("=");
-                if (formData.length == 2) {
-                    final String name = formData[0];
-                    final String preValue = formData[1];
-                    bodyMap.put(name, preValue);
-                } else {
-                    log.warn("Parse form data error: {}", element);
+        String[] splits = null;
+        HashMap<String, Object> bodyMap = null;
+
+        if (commandRequest.getBoundary().isEmpty()) {
```

Comment:
isEmpty가 아니라 Null 체크가 필요해 보이네요! 
아마 StringUtils.isEmpty 를 사용하시려고 하신것 같네요!

```suggestion
        if (StringUtils.isEmpty(commandRequest.getBoundary())) {
```

Reply:
동의합니다.